Python/search/fibonacci_search.py

```diff
@@ -0,0 +1,99 @@
+# Fibonacci search implementation in Python3
+
+"""
+Returns the index of a key in a sorted array or returns -1 if it is not contained
+"""
+
+def FibonacciSearch(arr, x, n):
+    """
+    Inputs-
+    arr: A sorted array in which we're going to search for the key
+    x: The number to be searched for
+    n: Size of the array
+
+    Output-
+    Single integer
+    The index of the key if it exists in the array.
+    else -1
+    """
+
+    # initialize fibonacci numbers
+    m2 = 0  # (m-2)'th Fibonacci No.
+    m1 = 1  # (m-1)'th Fibonacci No.
+    m = m2 + m1  # m'th Fibonacci
+
```

Comment:
Use proper naming conventions, Don't use single letter for declaring variables.

Reply:
I have modified it. Kindly let me know if there are any issues now!
@ankitaggarwal23 Please review this.